website/screens/components/resultset-table/code/examples/basicUsage.ts

```diff
@@ -13,42 +12,50 @@ const code = `() => {
     { displayValue: "Actions"},
   ];
 
+  const actions = [
+    {
+      icon: deleteIcon,
+      title: "icon",
+      onClick: () => {},
+    },
+  ];
+
   const rows = [
     [
       { displayValue: "001"},
       { displayValue: "Peter"},
       { displayValue: "Miami"},
-      { displayValue: <DxcButton icon={deleteIcon} /> },
+      { displayValue: <DxcResultsetTable.ActionsCell actions={actions} /> },
```

Comment:
One small detail to take into account, the button component can still be used inside the default sized table.
ActionsCell is only mandatory for the reduced table.